appveyor.yml

```diff
@@ -51,6 +51,7 @@ environment:
     - JOB: "Toolchain"
       GENERATOR: Visual Studio 14 2015 Win64
       CONFIGURATION: "Release"
+      ARROW_BUILD_GANDIVA: "OFF"
```

Comment:
Put this under "global" instead?

Reply:
I believe that installing llvmdev and building Gandiva is going to significantly increase build times. Once we get the build running for one entry in the matrix let's look at moving it to global

Reply:
I was thinking of moving the default of "OFF" to "global" and then overriding it where we see fit.

Reply:
Oh. Ok I'll change that